src/test/java/org/opendatakit/common/persistence/QueryResultTest.java

```diff
@@ -361,10 +363,21 @@ public void testCase3() throws ODKDatastoreException {
         int fetchSizes[] = {1, 2, 1021, 303, 101, 2831 };
         int idxFetch = 4;
         int len = 0;
-        QueryResult result = query.executeQuery(null, fetchSizes[idxFetch]);
+        // It seems that the following two asserts try to test if the results

```

Comment:
I think the comment could have been worded better. What about this?

```
// It seems that the following two asserts try to test if the results
// have less rows than the ones retrieved from the database by the query
// This was the original query:
// QueryResult result = query.executeQuery(null, fetchSizes[idxFetch]);
// This is the modified query ensuring that the page size is less than
// the number of rows in the database
```

Reply:
Sorry about that... `english.exe` had stopped working last night.